python/orca/src/bigdl/orca/learn/tf2/ray_estimator.py

```diff
@@ -417,6 +417,10 @@ def predict(self, data, batch_size=None, verbose=1,
         :param data_config: An optional dictionary that can be passed to data creator function.
         :param feature_cols: Feature column name(s) of data. Only used when data is a Spark
                DataFrame or an XShards of Pandas DataFrame. Default: None.
+        :param min_partition_num: Int. An optional param for repartition the input data when data
+               is an **orca.data.tf.data.Dataset**. If min_partition_num != None, the input data
+               will be repartitioned to max(min_partition_num, worker_num) partitions.
```

Comment:
Shall we add more guides about when and how to set `min_partition_num`, as well as the side effect (changed order of output compared to input)?